pytorch_lightning/trainer/auto_mix_precision.py

```diff
@@ -25,8 +25,9 @@ def init_amp(self):
         if self.use_amp and not APEX_AVAILABLE:  # pragma: no-cover
             raise ModuleNotFoundError(
                 "You set `use_amp=True` but do not have apex installed."
-                "Install apex first using this guide and rerun with use_amp=True:"
-                "https://github.com/NVIDIA/apex#linux his run will NOT use 16 bit precision"
+                " Install apex first using this guide: https://github.com/NVIDIA/apex#linux"
+                " and rerun with use_amp=True."
```

Comment:
```suggestion
                " and rerun with `use_amp=True`."
```
Looks good :) add single quotes for consistency with the first line (or remove them from there)